src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/FailedDocUploadProcessor.java

```diff
@@ -55,6 +56,7 @@ public FailedDocUploadProcessor(
         EnvelopeRepository envelopeRepository,
         ProcessEventRepository eventRepository,
         ContainerMappings containerMappings, // NOSONAR
+        OcrValidator ocrValidator, // NOSONAR
```

Comment:
inheritance 😢

Reply:
what does it say?

Reply:
Note that this job (faied doc upload) does not need this validator at all, but it's injected for the compatibility reasons.
Same as `ContainerMappings` and `ServiceBusHelper`